.github/workflows/deploy-do.yml

```
@@ -8,86 +8,101 @@
 jobs:
-    deploy-do:
-        runs-on: ubuntu-latest
+    deploy-dev:
+        runs-on: ubuntu-22.04
+
+        environment: dev
 
         steps:
             - name: Checkout
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)